### DISCUSSION TITLE
Repurpose spin bug into the Speen power/curse

### DISCUSSION THREAD

**Discussion by Leander-0** [unitystation/unitystation#7338] [Ideas]
god gives the silliest battles to his funniest clowns

### Description

 On staging 21082412 if you eat 2 pills of 5u with water and potassium your should  stomach explode like it happens on build 4121 but you instead manage to make spin any object at the top right tile of your character.

could be used for clown power like one of the mimes just to spin stuff like Waffielz suggested , or a magic state on a character that his curse is to spin the things near him.

https://user-images.githubusercontent.com/80334192/131230359-c1b7f8bd-68b8-4b68-9ad3-b60c54c4ab91.mp4

Steps to reproduce bug:
1. make 1 pill 5u potassium and 1 pill 5u water
2. eat both
3. enjoy the Speen™

**Comment by mmails** [comment]
Would be a fun addition if they were to add a clown spellbook or clown wizard garbs

**Comment by wraith-54321** [comment]
ehh, I am kinda thinking mime wizard, due to cloon ops being a thing

**Comment by PerfectTangent** [comment]
converting into a discussion.

**Comment by corp-0** [comment]
honestly I should move this back to issue and give it high priority
